Article/Article.js

```diff
@@ -3,18 +3,19 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector(".expandButton");
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = "expand";
     // Set a click handler on the expandButton reference, calling the expandArticle method.
+    this.expandButton.addEventListener("click", () => this.expandArticle());
```

Comment:
Great job using es6 syntax here. Very excited to see you using arrow syntax over the this keyword. Great job.